test/cli/cli.test.js

```diff
@@ -199,75 +197,9 @@ describe('CLI', () => {
     });
   });
 
-  it('should use different random port when multiple instances are started on different processes', (done) => {
-    const cliPath = resolve(__dirname, '../../bin/webpack-dev-server.js');
-    const simpleConfig = resolve(__dirname, '../fixtures/simple-config');
-
-    const cp = execa('node', [cliPath, '--colors=false'], {
-      cwd: simpleConfig,
-    });
-    const cp2 = execa('node', [cliPath, '--colors=false'], {
-      cwd: simpleConfig,
-    });
-
-    const runtime = {
-      cp: {
-        port: null,
-        done: false,
-      },
-      cp2: {
-        port: null,
-        done: false,
-      },
-    };
+  it('should use different random port when multiple instances are started on different processes', async () => {
```

Comment:
This test fails so I need to fix it.

Reply:
Oh, this test in the master branch is broken as well...

> cp1:null cp2: 8080

Yes this test passes because null !== 8080 but we don't expect it.

Reply:
I've fixed this test.